RecoBTag/Combined/python/DeepFlavourJetTagsProducer_cfi.py

```diff
@@ -0,0 +1,13 @@
+import FWCore.ParameterSet.Config as cms
+
+pfDeepCSVJetTags = cms.EDProducer(
```

Comment:
preferably, keep one producer per cfi file
and keep the same name for the file base as the producer name (e.g. pfDeepCSVJetTags_cfi and pfDeepCMVAJetTags_cfi)